test/library/tomography/test_process_tomography.py

```diff
@@ -100,7 +100,7 @@ def test_cvxpy_gaussian_lstsq_cx(self):
         )
```

Comment:
As due diligence, I must point out that no tests were added that accept the new arguments ``backend`` or``analysis`` here and in QST.